draft-ietf-shmoo-cancel-meeting.md

```diff
@@ -165,11 +167,11 @@ remote?
 
 # Remedies
 
-If the event cannot be held at the scheduled time and place, the IETF has
-several options. The remedies below should be consdered in light of these
-principles, presented in no particular order:
+If an event cannot be held at the scheduled time and place, the LLC and IESG
```

Comment:
```suggestion
If a meeting cannot be held at the scheduled time and place, the LLC and IESG
```

Reply:
Could you also check other uses of "event" vs. "meeting" throughout the document? A "meeting" is what we hold, an "event" is the thing that might cause us to cancel a meeting (at least I think that is how most of the text is written.)

Reply:
OK, it's done. If you're now happy with it, I'll merge this and publish a new version.